cmd/unused/internal/ui/table.go

```diff
@@ -48,9 +48,10 @@ func Table(ctx context.Context, options Options) error {
 	for _, d := range disks {
 		p := d.Provider()
 
-		row := []string{p.Name(), d.Name(), internal.Age(d.CreatedAt()), internal.Age(d.LastUsedAt())}
+		row := []string{p.Name(), d.Name(), internal.Age(d.CreatedAt()), internal.Age(d.LastUsedAt()), string(d.DiskType()), fmt.Sprintf("%d", d.SizeGB())}
 		for _, c := range options.ExtraColumns {
-			row = append(row, d.Meta()[c])
+			meta := d.Meta()
```

Comment:
Let's move this variable outside the `for` loop, we just need it once. I missed this before!